### PR TITLE
Feature#43/people description api

### PR DESCRIPTION
**Pull request by q270311**
PersonDetails component is only draft

### CODE REVIEW

File: src/features/personDetails/index.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    useEffect(() => {
+        dispatch(getDetailsForPerson({ personId: id }));
+    }, [id, dispatch]);
+    const status = useSelector(selectStatus);
+    const details = useSelector(selectDetails);
+    const cast = useSelector(selectCast);
+    const crew = useSelector(selectCrew);
+    const castNumber = (cast.filter(({ title }) => title)).length;
+    const crewtNumber = (crew.filter(({ title }) => title)).length;
```
Comment (kepkaklaudia) [inline src/features/personDetails/index.js:21]: `crewtNumber` - isn't it supposed to be `crewNumber`?

File: src/features/personDetails/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                    <b>{details.name}</b>
+                    <img src={`${secureBaseUrl}${details.profile_path}`} alt="" />
+                    <p>Date of birth:{details.birthday}</p>
+                    <p>Place of birth: {details.place_of_birth}</p>
+                    <p>{details.biography}</p>
+                    <b>Movies - cast ({castNumber})</b>
+                    {
+                        cast.map(film => <p>{film.title}</p>)
+                    }
+                    <b>Movies - crew ({crewtNumber})</b>
```
Comment (kepkaklaudia) [inline src/features/personDetails/index.js:38]: `crewtNumber` - isn't it supposed to be `crewNumber`?

**Review by kepkaklaudia — APPROVED** [review]
Great! There's not much left to finish 😄

File: src/features/personDetails/index.js (hunk 1 above)
Comment (q270311) [reply src/features/personDetails/index.js:21]: unfortunately not. movies and series are together in one table

File: src/features/personDetails/index.js (hunk 2 above)
Comment (q270311) [reply src/features/personDetails/index.js:38]: title form movie, name and media_type: "tv" for serial

File: src/features/personDetails/index.js (hunk 1 above)
Comment (kepkaklaudia) [reply src/features/personDetails/index.js:21]: I mean just the name of this constant. Why in `const crewtNumber` is there a "`t`" in the middle 🤔

Comment (panijoanna) [reply src/features/personDetails/index.js:21]: It's a good question, indeed it looks like a typo.

**Pull request merged**